learn/what_is_meilisearch/features.md

```diff
@@ -18,51 +18,48 @@ Meilisearch's default [relevancy rules](/learn/core_concepts/relevancy.md) are d
-Read more about synonyms in [this dedicated guide](/learn/configuration/synonyms.md).
-
-## Highlighting
+## Geosearch
 
-[Highlight](/reference/api/search.md#attributes-to-highlight) query terms so that matches pop out to the eye. Users don't need to read the entire text to find the match.
+Explore the physical world. [Geosearch](/learn/advanced/geosearch.md) allows you to filter and sort results based on their physical location.
 
 ## Filters
 
-Meilisearch allows you to define [filters](/learn/advanced/filtering_and_faceted_search.md) so you can filter through the results based on user-defined criteria.
+Define [filters](/learn/advanced/filtering_and_faceted_search.md) to refine results based on user-defined criteria.
 
 ## Faceting
 
-[Faceted search](/learn/advanced/filtering_and_faceted_search.md) allows you to classify search results into categories and to build intuitive navigation interfaces.
+Build intuitive navigation interfaces with [faceted search](/learn/advanced/filtering_and_faceted_search.md).
 
 ## Sorting
 
-[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which types of results they want to see first.
-
-## Placeholder search
+[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which results they want to see first.
 
-If you make a search without inputting any query words, Meilisearch will return all the documents in that index sorted by its [custom ranking rules](/learn/core_concepts/relevancy.md#custom-rules) and [sorting rules](/learn/advanced/sorting.md#sorting). This feature is called **placeholder search**.
+## API key management
 
-Placeholder searches are particularly effective when used with other features such as [faceting or filtering](/learn/advanced/filtering_and_faceted_search.md#filters-or-facets), which allow users to narrow their searches and browse by category. You can read more about this feature in our article on [search parameters](/reference/api/search.md#placeholder-search).
+[Protect your instance with API keys](/learn/security/master_api_keys.md). Control which users can access what indexes, routes, and endpoints.
 
-## Phrase search
+## Multitenancy
 
-If you enclose search terms in double quotes (`"`), Meilisearch will only return documents that contain those terms in the order they were given. This gives users the option to make more precise search queries.
+Manage complex multi-user applications handling sensitive data. [Tenant tokens](/learn/security/tenant_tokens.md) help you decide which documents each one of your users can search.
 
-Phrase search is particularly useful when looking for a single result, such as searching for a book by ISBN. It is also possible to combine phrase searches with Meilisearch's basic syntax so only parts of a query are matched strictly.
+## Synonyms
 
-## API key management
+[Define synonyms](/learn/configuration/synonyms.md) and craft a tailored, intuitive search experience.
```

Comment:
Removed the image because it felt a bit wild to have "Synonyms" as the only illustrated feature. The image also made it hard to push it down the feature order.

Since this is not exactly technical content, it does benefit from having images, IMHO. If we're going to keep it here on the long run, we should create a few.